service-map/src/main/resources/META-INF/OskariLayer.xml

```diff
@@ -551,16 +567,25 @@
 
     l.created,
     l.updated,
+   	gl.order_number,
 
     l.capabilities_last_updated,
     l.capabilities_update_rate_sec
 
     FROM
-    oskari_maplayer l LEFT OUTER JOIN oskari_maplayer_metadata m
-    ON  m.metadataid = l.metadataId
+    oskari_maplayer l
+    LEFT OUTER JOIN oskari_maplayer_metadata m ON  m.metadataid = l.metadataId
+    LEFT OUTER JOIN oskari_maplayer_group_link gl ON gl.maplayerid = l.id
+    <dynamic>
```

Comment:
This `<dynamic>` part should be removed. We just got rid of this join in #137. Nothing gets SELECTed from the oskari_maplayer_projections p table so it's easy to see that this isn't needed.